syft/frameworks/torch/tensors/interpreters/native.py

```diff
@@ -1028,6 +1031,15 @@ def encrypt(self, protocol="mpc", **kwargs):
 
             return x_encrypted
 
+        elif protocol.lower() == "fe":
+            scheme = kwargs.get("scheme")
+
+            x = self.copy()
```

Comment:
Do you think we have to copy?

Reply:
Not sure. Tbh just followed PaillierTensor but I guess I will remove the copy